Frameworks/CoreText/CTFont.mm

```diff
@@ -225,10 +283,8 @@ CFTypeRef CTFontCopyAttribute(CTFontRef font, CFStringRef attribute) {
 /**
  @Status Interoperable
 */
-float CTFontGetSize(CTFontRef font) {
-    float ret = [(UIFont*)font pointSize];
-
-    return ret;
+CGFloat CTFontGetSize(CTFontRef font) {
+    return font->_size;
```

Comment:
good point

---
In reply to: [78206516](https://github.com/Microsoft/WinObjC/pull/987#discussion_r78206516) [](ancestors = 78206516)